app/index.js

```diff
@@ -1,3 +0,0 @@
-import 'whatwg-fetch';
-// import data from the json file into index.js
-import './package.json';
```

Comment:
There is no `package` file to import. At least no in the `app` directory and nothing that has meaning to showing spotify data.